src/NodeJS/Server.php

```diff
@@ -194,6 +202,46 @@ public function getNodeModulesPath()
     }
 
     /**
+     * Get a single option.
+     *
+     * @param string $option The option name to retrieve.
+     * @param mixed $default_value The value to return if the option is not set.
+     *
+     * @return mixed The option value or default value if it is not set.
+     */
+    public function getOption($option, $default_value = NULL)
+    {
+        if (!isset($this->options[$option])) {
+          return $default_value;
+        }
+
+        return $this->options[$option];
+    }
+
+    /**
+     * Return the all options.
+     *
+     * @return array
+     */
+    public function getOptions()
+    {
+        return $this->options;
+    }
+
+    /**
+     * Set options array.
+     *
+     * @param array $options Array of options to set.
+     */
+    public function setOptions($options)
+    {
+        foreach ($options as $key => $value) {
+            $this->options[$key] = $value;
+        }
+        $this->serverPath = $this->createTemporaryServer();
```

Comment:
Please add empty line above. If you have any automated code style checks in place for PSR2 standard then this should have been reported as error by them. Also the Scrutinizer CI should have picked that as well.